Content.Client/Atmos/Overlays/AtmosDebugOverlay.cs

```diff
@@ -243,7 +242,7 @@ private void DrawTooltip(DrawingHandleScreen handle, Vector2 pos, AtmosDebugOver
 
         var moles = data.Moles == null
             ? "No Air"
-            : data.Moles.Sum().ToString();
+            : data.Moles.Sum().ToString(CultureInfo.InvariantCulture);
```

Comment:
What does this change?

Reply:
Rider code inspector says that string culture should be specified explicitly﻿

Reply:
It suggested 2 solutions: current culture and invariant culture. To avoid making a mistake I learnt a bit about these thing. This one uses float type, so Invariant culture should be used in this case

Reply:
My bad, I was wrong. CurrentCulture will output as "1,234.56". Invariant one just does not affect the value so it keeps it as "1.234,56".

> Are we making an explicit choice to express "1234.56" that way in space?

Yes, that way we'll keep the output as-is